spec/factories/application_form.rb

```diff
@@ -120,8 +120,8 @@
       further_information { Faker::Lorem.paragraph_by_chars(number: 300) }
       disclose_disability { %w[true false].sample }
       disability_disclosure { Faker::Lorem.paragraph_by_chars(number: 300) }
-      address_line2 { Faker::Address.city }
-      address_line3 { Faker::Address.county }
+      address_line3 { Faker::Address.city }
+      address_line4 { Faker::Address.county }
       address_line4 { '' }
```

Comment:
```suggestion
      address_line2{ '' }
      address_line3 { Faker::Address.city }
      address_line4 { Faker::Address.county }
```

Reply:
I forgot to force push and my fix hadn't come through. `address_line2` doesnt need to be set at all.